src/celeritas/em/model/SeltzerBergerModel.cc

```diff
@@ -154,30 +155,16 @@ void SeltzerBergerModel::step(CoreParams const&, CoreStateDevice&) const
 void SeltzerBergerModel::append_table(ImportSBTable const& imported,
                                       HostXsTables* tables) const
 {
-    auto reals = make_builder(&tables->reals);
+    CELER_EXPECT(imported);
 
-    CELER_ASSERT(!imported.value.empty()
-                 && imported.value.size()
-                        == imported.x.size() * imported.y.size());
     size_type const num_x = imported.x.size();
     size_type const num_y = imported.y.size();
 
     SBElementTableData table;
-
-    //! \todo Refactor as a builder helper using DedupeCollectionBuilder
-
-    // Incident charged particle log energy grid
-    table.grid.x = reals.insert_back(imported.x.begin(), imported.x.end());
-
-    // Photon reduced energy grid
-    table.grid.y = reals.insert_back(imported.y.begin(), imported.y.end());
-
-    // 2D scaled DCS grid
-    table.grid.values
-        = reals.insert_back(imported.value.begin(), imported.value.end());
+    table.grid = TwodGridBuilder{&tables->reals}(imported);
```

Comment:
I think we (I) still need to go back and refactor this as a builder helper class that can keep a persistent 2d grid builder across multiple elements.

Reply:
I went ahead and added this.